pkg/log/log.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2017 the u-root Authors. All rights reserved.
```

Comment:
not for now, but is this a good place to define a way to log 'progress', including the basic 'print progress on a SIGHUP'?